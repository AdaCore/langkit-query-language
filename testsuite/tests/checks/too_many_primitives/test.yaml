driver: 'checker'
rule_name: 'Too_Many_Primitives'
input_sources: ['foo.ads']
