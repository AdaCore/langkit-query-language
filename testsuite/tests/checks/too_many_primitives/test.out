foo.ads:2:9: error: tagged type has too many primitives
2 |    type T is tagged null record;
  |         ^ 

